crowbar_framework/app/models/provisioner_service.rb

```diff
@@ -126,8 +128,9 @@ def transition(inst, name, state)
     end
     if state == "hardware-installing"
       role = RoleObject.find_role_by_name "provisioner-config-#{inst}"
-      db = Proposal.where(barclamp: "provisioner", name: inst).first
-      add_role_to_instance_and_node("provisioner",inst,name,db,role,"provisioner-bootdisk-finder")
+      Proposal.where(barclamp: "provisioner", name: inst).first do |db|
+        add_role_to_instance_and_node("provisioner", inst, name, db, role, "provisioner-bootdisk-finder")
```

Comment:
Metrics/LineLength: Line is too long. [105/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)